2025-01-16 17:26:00,449 - INFO - Finished! Total sfe time: 5.354297399520874 
2025-01-16 17:26:00,450 - DEBUG - 12,server, Parsed server ram usage: 79868
2025-01-16 17:26:00,450 - DEBUG - 12,client, Parsed client ram usage: 80136
2025-01-16 17:26:15,164 - DEBUG - client all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 6 second(s) each.', '2024-12-18 21:23:43,268 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.1:61002', '2024-12-18 21:23:43,269 - tno.mpc.communication.httphandlers - ERROR - Message not received.', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1061, in _wrap_create_connection', '    sock = await aiohappyeyeballs.start_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 103, in start_connection', '    raise first_exception', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 81, in start_connection', '    sock = await _connect_sock(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 165, in _connect_sock', '    await loop.sock_connect(sock, address)', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 502, in sock_connect', '    return await fut', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 537, in _sock_connect_cb', "    raise OSError(err, f'Connect call failed {address}')", "ConnectionRefusedError: [Errno 111] Connect call failed ('192.168.1.2', 61001)", '', 'The above exception was the direct cause of the following exception:', '', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/tno/mpc/communication/httphandlers.py", line 193, in _send', '    async with self.session.post(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 1355, in __aenter__', '    self._resp: _RetType = await self._coro', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 659, in _request', '    conn = await self._connector.connect(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 557, in connect', '    proto = await self._create_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1002, in _create_connection', '    _, proto = await self._create_direct_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1336, in _create_direct_connection', '    raise last_exc', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1305, in _create_direct_connection', '    transp, proto = await self._wrap_create_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1076, in _wrap_create_connection', '    raise client_error(req.connection_key, exc) from exc', "aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host 192.168.1.2:61001 ssl:default [Connect call failed ('192.168.1.2', 61001)]", '2024-12-18 21:23:44,316 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-18 21:23:44,339 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-18 21:23:44,408 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-18 21:23:44,436 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', 'TP is  0.001', '2024-12-18 21:23:44,446 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-18 21:23:44,459 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-18 21:23:44,460 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.1:61002 shutdown', 'Total bytes received: 1930', 'Total messages received: 5', '2024-12-18 21:23:44,462 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.2:61001 shutdown', 'Total bytes sent: 766', 'Total messages sent: 5', '2024-12-18 21:23:44,462 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 766', 'Total messages sent: 5', 'Total bytes received: 1930', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 1 12"', '\tUser time (seconds): 3.67', '\tSystem time (seconds): 0.48', '\tPercent of CPU this job got: 79%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:05.22', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80328', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 16079', '\tVoluntary context switches: 66', '\tInvoluntary context switches: 275', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_rpi_2025-01-16_17-25-55.csv']
2025-01-16 17:26:15,164 - DEBUG - server all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 6 second(s) each.', '2024-12-18 21:24:20,572 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.2:61001', '2024-12-18 21:24:21,538 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-18 21:24:21,630 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-18 21:24:21,645 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-18 21:24:21,673 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', 'TP is  0.001', '2024-12-18 21:24:21,700 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-18 21:24:21,703 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-18 21:24:21,705 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.2:61001 shutdown', 'Total bytes received: 766', 'Total messages received: 5', '2024-12-18 21:24:21,706 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.1:61002 shutdown', 'Total bytes sent: 1930', 'Total messages sent: 5', '2024-12-18 21:24:21,707 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 1930', 'Total messages sent: 5', 'Total bytes received: 766', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 0 12"', '\tUser time (seconds): 3.67', '\tSystem time (seconds): 0.50', '\tPercent of CPU this job got: 79%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:05.22', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80504', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 15964', '\tVoluntary context switches: 67', '\tInvoluntary context switches: 129', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_rpi_2025-01-16_17-25-55.csv']
2025-01-16 17:26:15,901 - DEBUG - All values from powertop for client: 1       168 mW  
11     13.3 mW  
12     12.3 mW  
Name: 7, dtype: object
2025-01-16 17:26:15,901 - DEBUG - Energy client: 0.9002400000000002
2025-01-16 17:26:16,676 - DEBUG - All values from powertop for server: 1       398 mW  
12     16.6 mW  
18     8.36 mW  
Name: 7, dtype: object
2025-01-16 17:26:16,676 - DEBUG - Energy server: 1.9456159999999998
